x-pack/elastic-agent/magefile.go

```diff
@@ -523,7 +523,7 @@ func packageAgent(requiredPackages []string, packagingFn func()) {
 		defer os.RemoveAll(dropPath)
 		defer os.Unsetenv(agentDropPath)
 
-		packedBeats := []string{"filebeat", "metricbeat"}
+		packedBeats := []string{"filebeat", "heartbeat", "metricbeat"}
```

Comment:
@blakerouse Do we directly have to bundle it to play around with it or would download work?

Reply:
@ruflin We do have to bundle it to work with it from master. On a SNAPSHOT build it would be able to download. I think its best to bundle it, as we are going to now bundle Endpoint as well.

Reply:
This helps air gapped users right?

Reply:
This helps developers that use `mage package` inside of `x-pack/elastic-agent` to build Elastic Agent when heartbeat bundled. Which you want to do to easy testing.

Reply:
I'm good with bundle it at the moment in master to move forward but we should have a separate discussion if we want to package it or not as we did for other other binaries.